models/library.py

```diff
@@ -74,6 +79,28 @@ def return_time(self, dt):
             raise ValueError("return_time has to be timezone aware")
         self._return_time = dt.astimezone(tz=pytz.utc)
 
+    @property
+    def reservation_begin(self):
+        return self._reservation_begin.replace(tzinfo=pytz.utc). \
+            astimezone(tz=pytz.timezone('Europe/Warsaw'))
+
+    @return_time.setter
+    def reservation_begin(self, dt):
+        if dt.tzinfo is None:
+            raise ValueError("reservation_begin has to be timezone aware")
+        self._reservation_begin = dt.astimezone(tz=pytz.utc)
+
+    @property
+    def reservation_end(self):
+        return self._reservation_end.replace(tzinfo=pytz.utc). \
+            astimezone(tz=pytz.timezone('Europe/Warsaw'))
+
+    @return_time.setter
+    def reservation_end(self, dt):
+        if dt.tzinfo is None:
+            raise ValueError("reservation_end has to be timezone aware")
+        self._reservation_end = dt.astimezone(tz=pytz.utc)
```

Comment:
I think `reservation_end` should be calculated on the basis of `reservation_begin` field.